### PR TITLE
touch build-started with bin/start-build

### PR DESCRIPTION
**Pull request by davidsbailey**
Slightly fudged command line output:

```
ubuntu@staging:~/staging$ ./bin/start-build 
touch /home/ubuntu/staging/build-started

ubuntu@staging:~/staging$ ./bin/start-build 
./start-build:7:in `block in <main>': file 'build-started' already exists (RuntimeError)
    from ./start-build:6:in `chdir'
    from ./start-build:6:in `<main>'

Davids-MacBook-Pro-2:~/src/cdo/bin (start-build)$  ./bin/start-build 
./start-build:4:in `<main>': ./start-build does not work in development mode (RuntimeError)
```

### CODE REVIEW

File: bin/start-build [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+require_relative '../deployment.rb'
+
+raise "#{$0} does not work in development mode" if rack_env?(:development)
+
+Dir.chdir(deploy_dir) do
+  raise "file 'build-started' already exists" if File.file?('build-started')
+  puts "touch #{`pwd`.chomp}/build-started"
```
Comment (aoby) [inline bin/start-build]: Why not Dir.pwd?

**Comment by aoby** [comment]
Is this an improvement over touching build-started directly? What problem is this trying to solve?

Please add a small description at the top explaining what this is for. I realize start_build is fairly self-explanatory, but it's not necessarily obvious that this is for triggering the build on a deployed environment, and it may take a minute for the cronjob to notice.

**Comment by aoby** [comment]
otherwise lgtm

**Comment by davidsbailey** [comment]
@aoby good questions. The problems this is trying to solve is that "touch build-started" is (1) undiscoverable (2) people touch the wrong file and then wait while nothing happens (3) people "touch build-started" when the file is already there and then wait while nothing happens (because the build is stuck for another reason). 

 I've also added a hipchat log message so that the caller will see output to the appropriate room when the command is successfully called.

This solution is definitely just an incremental improvement and leave some things to be desired (e.g. no 1-minute delay waiting for cron job). If anyone knows a way to fix this properly/better I'm all ears.

**Comment by Bjvanminnen** [comment]
cool. thanks for doing this dave :)

**Pull request merged**